float64 distance
float64 bearing
float32 rel_alt